code/modules/clothing/chameleon.dm

```diff
@@ -580,6 +580,41 @@
 	var/datum/action/item_action/chameleon/drone/randomise/randomise_action = new(src)
 	randomise_action.UpdateButtonIcon()
 
+/datum/action/item_action/chameleon/tongue_change
+	name = "Tongue Change"
+	icon_icon = 'icons/obj/surgery.dmi'
+	button_icon_state = "tonguebone"
+	var/list/tongue_list = list()
+
+/datum/action/item_action/chameleon/tongue_change/proc/generate_tongue_list()
+	for(var/V in typesof(/obj/item/organ/tongue))
+		if(ispath(V))
```

Comment:
negate this and continue early. Also, how can anything from types of not be a path?